### PR TITLE
Fix error messages

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR fixes messages for the errors happening while creating a new workspace, or updating an existing one.

### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/19945
provides a clearer error message for https://github.com/eclipse/che/issues/19922

### CODE REVIEW

File: src/store/Workspaces/index.ts [hunk 1]
```diff
@@ -209,24 +209,7 @@ export const actionCreators: ActionCreators = {
       }
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
-
-      const errorMessage = e?.message || '';
```
Comment (sleshchenko) [inline src/store/Workspaces/index.ts:213]: it's already a part of some helper method we have, right?

**Review by sleshchenko — APPROVED** [review]
Tested and did not face `undefined` in the error messages.

File: src/store/Workspaces/index.ts (hunk 1 above)
Comment (akurinnoy) [reply src/store/Workspaces/index.ts:213]: that's correct

Comment (olexii4) [reply src/store/Workspaces/index.ts:213]: Our [helper](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/helpers/getErrorMes) doesn't cover the next line:
```
 const responseMessage = e?.response?.data?.message || '';
 ```
 @akurinnoy 
 Do we need to add this structure to our helper?

**Review by JPinkney — APPROVED** [review]
LGTM

File: src/store/Workspaces/index.ts (hunk 1 above)
Comment (akurinnoy) [reply src/store/Workspaces/index.ts:213]: that makes sense, I'm preparing update

**Comment by sleshchenko** [comment]
[dashboard-ci-test]

**Review by olexii4 — APPROVED** [review]

File: src/store/Workspaces/index.ts (hunk 1 above)
Comment (olexii4) [reply src/store/Workspaces/index.ts:213]: Thanks

**Comment by sleshchenko** [comment]
[dashboard-ci-test]

**Pull request merged**